### PR TITLE
Create a suite to run cluster loader test from perf-tests repo

### PR DESCRIPTION
**Pull request by wojtek-t**
I'm not sure that it will work as is, but iterating on this seems to be the easiest thing to do.

### CODE REVIEW

File: jobs/ci-perf-tests-e2e-gce.env [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### job-env
+JENKINS_USE_GET_KUBE_SCRIPT=y
+PROJECT=???
```
Comment (krzyzacy) [inline jobs/ci-perf-tests-e2e-gce.env]: please fill

File: jobs/ci-perf-tests-e2e-gce.env [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+KUBE_NODE_OS_DISTRIBUTION=gci
+KUBEKINS_TIMEOUT=???
```
Comment (krzyzacy) [inline jobs/ci-perf-tests-e2e-gce.env]: please fill

File: kubetest/main.go [hunk 3]
```diff
@@ -88,6 +89,7 @@ func defineFlags() *options {
 	flag.BoolVar(&o.federation, "federation", false, "If true, start/tear down the federation control plane along with the clusters. To only start/tear down the federation control plane, specify --deploy=none")
 	flag.BoolVar(&o.kubemark, "kubemark", false, "If true, run kubemark tests.")
 	flag.BoolVar(&o.multipleFederations, "multiple-federations", false, "If true, enable running multiple federation control planes in parallel")
+	flag.BoolVar(&o.perfTests, "perf-tests", false, "If true, run perf tests.")
```
Comment (gmarek) [inline kubetest/main.go]: If true, run tests from perf-tests repo

File: prow/config.yaml [hunk 4]
```diff
@@ -1063,3 +1063,39 @@ periodics:
...
       name: cache-ssd
+
+- name: ci-perf-tests-e2e-gce-clusterloader
+  interval: 30m
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170418-c08e1094
+      args:
+      - "--repo=k8s.io/kubernetes=master"
+      - "--repo=k8s.io/perf-tests=master"
```
Comment (krzyzacy) [inline prow/config.yaml]: you'll probably also need: --timeout=blah

File: jobs/ci-perf-tests-e2e-gce.env (hunk 1 above)
Comment (wojtek-t) [inline jobs/ci-perf-tests-e2e-gce.env]: done

File: jobs/ci-perf-tests-e2e-gce.env (hunk 2 above)
Comment (wojtek-t) [inline jobs/ci-perf-tests-e2e-gce.env]: done

File: prow/config.yaml [hunk 5]
```diff
@@ -1063,3 +1063,39 @@ periodics:
...
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      - name:  JENKINS_GCE_SSH_PRIVATE_KEY_FILE
+        value: /etc/ssh-key-secret/ssh-private
+      - name:  JENKINS_GCE_SSH_PUBLIC_KEY_FILE
+        value: /etc/ssh-key-secret/ssh-public
```
Comment (krzyzacy) [inline prow/config.yaml]: you might not need those ssh entries. But if you do, you'll also need to add an USER:prow entry.

File: kubetest/main.go (hunk 3 above)
Comment (wojtek-t) [inline kubetest/main.go]: done

File: jobs/config.json [hunk 6]
```diff
@@ -3093,6 +3093,16 @@
     "--env-file=jobs/ci-kubernetes-e2e-gce-ubuntu-slow.env",
     "--mode=local"
   ]
+},
+
+"ci-perf-tests-e2e-gce-clusterloader": {
+  "scenario": "kubernetes_e2e",
+  "args": [
+    "--env-file=platforms/gce.env",
+    "--env-file=jobs/ci-perf-tests-e2e-gce.env",
```
Comment (krzyzacy) [inline jobs/config.json]: rename the env file to `ci-perf-tests-e2e-gce-clusterloader.env`

File: prow/config.yaml (hunk 4 above)
Comment (wojtek-t) [inline prow/config.yaml]: done

**Comment by krzyzacy** [comment]
you'll also need to make a new kubekins-e2e image for the kubetest/e2e-runner changes

File: testgrid/config/config.yaml [hunk 7]
```diff
@@ -816,6 +816,9 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/periodic-kubernetes-bazel-build-1-6
 - name: metrics-bigquery
   gcs_prefix: kubernetes-jenkins/logs/metrics-bigquery
+# perf tests
+- name: ci-perf-tests-e2e-gce-clusterloader
+  gce_prefix: kubernetes-jenkins/logs/ci-perf-tests-e2e-gce-clusterloader
```
Comment (krzyzacy) [inline testgrid/config/config.yaml]: bah... `gcs_prefix`...

Comment (wojtek-t) [inline testgrid/config/config.yaml]: fixed - thanks for pointing

**Comment by wojtek-t** [comment]
I built the new kubekins image: "v20170504-79009d67"

As @krzyzacy suggested, we will wait for the next run of:
https://k8s-testgrid.appspot.com/test-infra#prow-canary

to see if nothing was broken.

**Comment by krzyzacy** [comment]
/lgtm

I'll gonna bounce, feel free to merge once the canary job is passing, guess also keep an eye on other jobs.

**Comment by wojtek-t** [comment]
It seems that the last run (started 30 minutes after I built the new image) is green:
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-e2e-prow-canary/15755

So I'm merging this PR, in case of issues, I will revert it.

**Pull request merged**